ccle_tasks/data/stjude_rms/urls.txt

```diff
@@ -1,55 +0,0 @@
-/seq/picard_aggregation/RP-1561/Exome/BC-3C/current/BC-3C.bam
```

Comment:
we might want to keep this one.. just in case someone wants to know where the file came from

Reply:
resolved in new PR